src/products/mozilla_org/testing.md

```diff
@@ -102,14 +99,14 @@ The project dashboard contains fewer files than before. This list will grow ove
 ### Reporting technical issues
 
 Report issues that can’t be resolved in Pontoon by:
-
 * Filing a bug in [Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla%20Localizations) and selecting your locale.
 * Adding the l10n PM in charge for NI. The l10n PM information appears at the top left corner on the [project page](https://pontoon.mozilla.org/projects/mozillaorg/).
 * Including the file name plus a screenshot and/or URL.
+* If additional tag must be added in your language for a particular string in order to make it make sense, this will generate a linter error. You can request the l10n PM to make an exception.
```

Comment:
```suggestion
* If markup needs to change in your language for a particular string, this will generate a linter error. You can request the l10n PM to make an exception.
```

Do they need to request it though? The PM will see the error and ask.

Reply:
Reworded this section.